library/core/src/ptr/unique.rs

```diff
@@ -32,6 +32,8 @@ use crate::ptr::NonNull;
 )]
 #[doc(hidden)]
 #[repr(transparent)]
+// Lang item used experimentally by Miri to define the semantics of `Unique`.
+#[lang = "ptr_unique"]
```

Comment:
```suggestion
#[cfg_attr(not(bootstrap), lang = "ptr_unique")]
```
should fix the CI failure